cumulus/parachains/runtimes/assets/asset-hub-polkadot/src/lib.rs

```diff
@@ -615,6 +617,11 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type PriceForSiblingDelivery = NoPriceForMessageDelivery<ParaId>;
 }
 
+impl cumulus_pallet_xcmp_queue::migration::v4::V4Config for Runtime {
+	// This must be the same as the `ChannelInfo` from the `Config`:
```

Comment:
```suggestion
	/// This must be the same as the `ChannelInfo` from the `Config`:
```

Reply:
Its in an impl block, the trait should already describe it.